src/config/config.cc

```diff
@@ -152,6 +153,7 @@ Config::Config() {
       {"pidfile", true, new StringField(&pidfile, kDefaultPidfile)},
       {"max-io-mb", false, new IntField(&max_io_mb, 0, 0, INT_MAX)},
       {"max-bitmap-to-string-mb", false, new IntField(&max_bitmap_to_string_mb, 16, 0, INT_MAX)},
+      {"max-scan-num", false, new IntField(&max_scan_num, 2, 0, INT_MAX)},
```

Comment:
Seems `2` is too smaller to be as the `max_scan_num`?

Reply:
it's just for test easy，jihuayu say they will determine the final value.